src/applications/pensions/config/form.js

```diff
@@ -15,7 +14,6 @@ import bankAccountUI from 'platform/forms/definitions/bankAccount';
 import { VA_FORM_IDS } from 'platform/forms/constants';
 
 import currentOrPastDateUI from 'platform/forms-system/src/js/definitions/currentOrPastDate';
```

Comment:
There are a bunch of linting errors for deprecation of Importing from platform via platform/utilities